src/main/java/com/butbetter/storage/file_upload/FileStorageService.java

```diff
@@ -232,10 +232,9 @@ public void deleteAll() throws StorageException {
 				}
 			});
 		} catch (IOException e) {
-			String message = "couldn't remove all files, either because " +
+			String message = "skipping deleting all current existing files, couldn't remove all files, either because " +
 					"of permission problems or the path doesn't exist anymore";
 			logger.error(message, e);
-			throw new StorageException(message, e);
```

Comment:
The solution of the bug is removing from StorageException()

Reply:
This ensures, that the error will still be logged

initially, the idea of this method was to remove the temporary csv file dir, where the data will be temporarily stored before importing it into the Database

It was never meant to critically fail on not finding the directory, as I though it would just ignore the not existing directory instead of just fully throwing an IOException

now the deletion of the directory will still fail, but instead of stopping the initialization completely and therefore the running Program as well, it just says "didn't remove the directory because it's not there or I can't"

the removal isn't important enough to just die on trying it